services/users/utils.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package users
+
+import (
+	"strings"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/utils"
+)
+
+func (us *UserService) IsPasswordValid(password string) *model.AppError {
+
+	if *us.config().ServiceSettings.EnableDeveloper {
+		return nil
+	}
+
+	return utils.IsPasswordValidWithSettings(password, &us.config().PasswordSettings)
```

Comment:
This is a great opportunity to move this code from utils package to inside the users package. I see it only called in a single place. This is good stuff. :+1: